src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -147,10 +205,14 @@ public function getAccessibleWorkspaceNames()
      */
     public function getNode($path)
     {
+        if (empty($this->caches['nodes'])) {
+            return parent::getNode($path);
```

Comment:
actually misread this .. I made this change since in theory a user can have no caching for nodes active.